src/compute/flexbox.rs

```diff
@@ -23,6 +23,34 @@ use crate::tree::LayoutTree;
 #[cfg(feature = "debug")]
 use crate::debug::NODE_LOGGER;
 
+/// The public interface to Taffy's Flexbox algorithm implementation
+pub(crate) struct FlexboxAlgorithm;
```

Comment:
It definitely feels weird that this isn't fully pub, given the comment above.

Reply:
This one is actually fixed in #326. I'd probably rather leave that change there if that's ok. Because this PR isn't currently changing our public API at all, whereas #326 is already making significant changes to the public API already.

Reply:
Great that's fine by me.